sound/soc/intel/boards/bdw-rt5677.c

```diff
@@ -446,6 +446,7 @@ static struct platform_driver bdw_rt5677_audio = {
 	.probe = bdw_rt5677_probe,
 	.driver = {
 		.name = "bdw-rt5677",
+		.pm = &snd_soc_pm_ops
```

Comment:
@plbossart Hmm, I'm a bit confused why here it's ok to set the pm ops without #ifdefs , but in other machines drivers used by both SST and SOF, it's not. I'm probably missing something...

Reply:
The Atom/SST driver has its own power management sequences that I frankly don't understand well, we don't want to interfere with it. For Broadwell, there is no known issue so we can set this unconditionally.